internal/refreshtoken/item.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package refreshtoken
+
+import (
+	"time"
+
+	"github.com/hashicorp/boundary/internal/db/timestamp"
+	"github.com/hashicorp/boundary/internal/types/resource"
+)
+
+type item struct {
```

Comment:
Yes, it does implement pagination.Item. I normally like the implementation of the interface to be implicit, since this is only used inside our application and any missing functionality would be obvious, but I'll add it to the test file, let me know if that isn't sufficient.

Reply:
Ah, actually, asserting this creates an import cycle, so no can do 😬. I don't think it's strictly necessary, though.